dvc/proc/manager.py

```diff
@@ -63,17 +65,62 @@ def spawn(self, args: Union[str, List[str]], name: Optional[str] = None):
             pid,
         )
 
-    def send_signal(self, name: str, signal: int):
-        """Send `signal` to the specified named process."""
-        raise NotImplementedError
-
-    def kill(self, name: str):
-        """Kill the specified named process."""
-        raise NotImplementedError
-
-    def terminate(self, name: str):
-        """Terminate the specified named process."""
-        raise NotImplementedError
+    if sys.platform == "win32":
+
+        def send_signal(self, name: str, sig: int):
+            """Send `signal` to the specified named process."""
+            # pylint: disable=no-member
+            process_info = self[name]
+            if process_info.returncode is None:
+                return
+            if sig == signal.SIGTERM:
+                self.terminate(name)
+            elif sig == signal.CTRL_C_EVENT:
+                os.kill(process_info.pid, signal.CTRL_C_EVENT)
+            elif sig == signal.CTRL_BREAK_EVENT:
+                os.kill(process_info.pid, signal.CTRL_BREAK_EVENT)
```

Comment:
```suggestion
            if sig in (signal.SIGTERM, signal.SIGKILL):
                self.terminate(name)
            elif sig in (signal.CTRL_C_EVENT, signal.CTRL_BREAK_EVENT):
                os.kill(process_info.pid, sig)
```

Reply:
I edited my change here, I thought `signal.SIGKILL` was aliased to `SIGTERM` on windows in python, but it's not defined at all.

Reply:
I copied most of these codes from `Popen`, if we can safely use `os.kill` here I think we don't need this complex logic at all.

Reply:
For windows we should be checking the signal value though.

The issue being that if you use `os.kill()` on windows without checking the signal value, it will terminate the process on any signal other than `CTRL_C_EVENT` and `CTRL_BREAK_EVENT`. I think we should error out on the unexpected values instead of always terminating the process.